slides/syntroduction/Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: all
+all: slides.html slides.pdf
+
+.PHONY: serve
+serve: slides.html
+	caddy
```

Comment:
Using caddy _v2.1.1_, caddy needs the subcommand `run` to run in the foreground or `start` to start as daemon.

Reply:
I haven't put any work into the plumbing, it's plain 1:1 copy of our internal template and I actually focused on the content. If this needs to be fixed, we should do it in our template and update this here when there is a need.